src/doc/intro.md

```diff
@@ -392,20 +392,23 @@ Here's an example of a concurrent Rust program:
 use std::thread::Thread;
 
 fn main() {
-    for _ in range(0u, 10u) {
-        Thread::spawn(move || {
+    let guards: Vec<_> = (0..10).map(|_| {
+        Thread::scoped(move || {
```

Comment:
In this case due to not closing over anything you can actually remove the `move` keyword.